modules/sms_email_reminder/cron_functions.php

```diff
@@ -1,148 +1,76 @@
-        }
-
-        $cnt = "";
-        $cnt .= "\nHeaders : " . $headers;
-        $cnt .= "\nDate Time :" . date("d M, Y  h:i:s");
-        $cnt .= "\nTo : " . $to;
-        $cnt .= "\nSubject : " . $subject;
-        $cnt .= "\nBody : \n" . $vBody . "\n";
-
-        if (1) {
-            //WriteLog($cnt);
-        }
-
+    if ($GLOBALS['SMTP_HOST'] == '') {
         $mstatus = true;
-        $mstatus = @mail($to, $subject, $vBody, $headers);
-        // larry :: debug
-        //echo "\nDEBUG :email: send email from=".$from." to=".$to." sbj=".$subject." body=".$vBody." head=".$headers."\n";
-        //echo "\nDEBUG :email: send status=".$mstatus."\n";
+        $mstatus = @mail($to, $cc, $subject, $vBody);
     } else {
-        // larry :: debug
-        //echo "\nDEBUG :: use smtp method\n";
-
-        if (!class_exists("smtp_class")) {
-            include("../../library/classes/smtp/smtp.php");
-            include("../../library/classes/smtp/sasl.php");
-        }
-
-        $strFrom = $from;
-        $sender_line = __LINE__;
-        $strTo = $to;
-        $recipient_line = __LINE__;
-        if (strlen($strFrom) == 0) {
-            return( false );
-        }
-
-        if (strlen($strTo) == 0) {
-            return( false );
-        }
-
-        //if( !$smtp )
-        $smtp = new smtp_class();
-
-        $smtp->host_name = $GLOBALS['smtp_host_name'];
-        $smtp->host_port = $GLOBALS['smtp_host_port'];
-        $smtp->ssl = $GLOBALS['smtp_use_ssl'];
-        $smtp->localhost = $GLOBALS['smtp_localhost'];
-        $smtp->direct_delivery = 0;
-        $smtp->timeout = 10;
-        $smtp->data_timeout = 0;
-
-        $smtp->debug = 1;
-        $smtp->html_debug = 0;
-        $smtp->pop3_auth_host = "";
-
-        $smtp->user = $GLOBALS['smtp_auth_user'];
-        $smtp->password = $GLOBALS['smtp_auth_pass'];
-
-        $smtp->realm = "";
-        // Workstation name for NTLM authentication
-        $smtp->workstation = "";
-        // Specify a SASL authentication method like LOGIN, PLAIN, CRAM-MD5, NTLM, etc..
-        // Leave it empty to make the class negotiate if necessary
-        $smtp->authentication_mechanism = "";
-
-        // If you need to use the direct delivery mode and this is running under
-        // Windows or any other platform
-        if ($smtp->direct_delivery) {
-            if (!function_exists("GetMXRR")) {
-                $_NAMESERVERS = array();
-                include("getmxrr.php");
+        if (!class_exists("SMTP")) {
+            $SenderName = $GLOBALS['patient_reminder_sender_name'];
+            $SenderEmail = $GLOBALS['patient_reminder_sender_email'];
+            if (!class_exists('PHPMailer\PHPMailer\PHPMailer')) {
+                require(__DIR__ . "/../../library/classes/PHPMailer/src/Exception.php");
+                require(__DIR__ . "/../../library/classes/PHPMailer/src/PHPMailer.php");
+                require(__DIR__ . "/../../library/classes/PHPMailer/src/SMTP.php");
```

Comment:
this block of requires does not look right. these files are not in core and the autoloader should have PHPMailer\PHPMailer\PHPMailer

Reply:
Sorry Brady, I know it doesn't look good. But I don't know how to solve it.

Reply:
rec removing the entire if !class_exists("SMTP) code block and just keeping:
```
            $SenderName = $GLOBALS['patient_reminder_sender_name'];
            $SenderEmail = $GLOBALS['patient_reminder_sender_email'];
```

Reply:
Then the `new PHPMailer()` should work just fine since you have the `use PHPMailer\PHPMailer\PHPMailer` at top